_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,927 @@
+                                      ]
+                                    },
+                                    {
+                                      "value": 2.0,
+                                      "description": "avgdl, average length of field",
+                                      "details": [
+                                        
+                                      ]
+                                    }
+                                  ]
+                                }
+                              ]
+                            }
+                          ]
+                        }
+                      ]
+                    },
+                    {
+                      "value": 0.0,
+                      "description": "Not a match",
+                      "details": [
+                        
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        "inner_hits": {
+          "location": {
+            "hits": {
+              "total": {
+                "value": 0,
+                "relation": "eq"
+              },
+              "max_score": null,
+              "hits": [
+                
+              ]
+            }
+          },
+          "user": {
+            "hits": {
+              "total": {
+                "value": 2,
+                "relation": "eq"
+              },
+              "max_score": 0.31506687,
+              "hits": [
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 0
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Wick",
+                    "age": 46
+                  }
+                },
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 1
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Snow",
+                    "age": 40
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## Sorting and Pagination with inner hits
+1. To apply sorting, users can add `sort` sub-clause in the `inner_hits` clause.
+2. To apply pagination users can pass `from` and `size` parameters in the `inner_hits` clause.
+3. Users can also define custom name for the inner hits in the search response. This is useful in differentiating between multiple inner hits in a single query.
+To know more about it, please read documentation [Inner hits parameter]({{site.url}}{{site.baseurl}}/search-plugins/searching-data/inner-hits/#inner_hits-parameters)
```

Comment:
can we have one more example with some of those features and inner hits, e.g. sorting and custom name